code/parse/sexp.cpp

```diff
@@ -899,6 +899,45 @@ sexp_ai_goal_link Sexp_ai_goal_links[] = {
 
 SCP_vector<dynamic_sexp_enum_list> Dynamic_enums;
 
+SCP_vector<dynamic_sexp_parameter_list> Dynamic_parameters;
+
+int get_dynamic_parameter_index(SCP_string op_name, int param)
+{
+	for (int i = 0; i < (int)Dynamic_parameters.size(); i++) {
+		if (!stricmp(Dynamic_parameters[i].operator_name.c_str(), op_name.c_str())) {
+
+			// Make sure there are actually parameters for this operator
+			if ((int)Dynamic_parameters[i].parameter_map.size() > 0) {
```

Comment:
No need to cast to int, if it returns a uint or size_t and you are checking if it's grater than zero.  Also, does parameter_map have an empty() method?  I'm unable to check.

Reply:
done